cmd/zqd/listen/command.go

```diff
@@ -34,5 +37,30 @@ func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 }
 
 func (c *Command) Run(args []string) error {
-	return zqd.Run(c.listenAddr)
+	logger := newLogger()
+	handler := zqd.NewHandler()
+	ln, err := net.Listen("tcp", c.listenAddr)
+	if err != nil {
+		return err
+	}
+	logger.Info("Listening", zap.String("addr", ln.Addr().String()))
+	return http.Serve(ln, handler)
+}
+
+func newLogger() *zap.Logger {
+	encoder := zap.NewProductionEncoderConfig()
+	encoder.CallerKey = ""
+	c := zap.Config{
+		Level:            zap.NewAtomicLevelAt(zap.DebugLevel),
+		Development:      false,
+		Encoding:         "json",
```

Comment:
Why json by default?

Reply:
Structured logs are and easily readable by jq! But also for the brim app I was going to have the os grab a random open port to use. I was going to parse the output from stdout so the client app can know which port was selected.